### PR TITLE
feat(DTNAVYA-1153): Add logic to show tagline for multiple buttons

### PR DESCRIPTION
**Pull request by lucasfrotam**
This PR is about the Upfront Presentment project and adds a new queryParam so we can show the tagline for multiple buttons using Personalization Studio.

https://engineering.paypalcorp.com/jira/browse/DTNAVYA-1153

### CODE REVIEW

**Review by lucasfarias2 — APPROVED** [review]
LGTM

File: server/components/buttons/middleware.js [hunk 1]
```diff
@@ -68,7 +68,7 @@ export function getButtonMiddleware({
 
             tracking(req);
 
-            const { env, clientID, buttonSessionID, cspNonce, debug, buyerCountry, disableFunding, disableCard, userIDToken, amount,
+            const { env, clientID, buttonSessionID, cspNonce, debug, buyerCountry, disableFunding, disableCard, userIDToken, amount, buttonsArray,
```
Comment (ssono) [inline server/components/buttons/middleware.js:71]: Lets make to update these fields to match whatever the other PR decides for the name. likely `renderedButtons`

File: server/components/buttons/params.js [hunk 2]
```diff
@@ -45,7 +45,8 @@ type ButtonInputParams = {|
     paymentMethodNonce? : ?string,
     paymentMethodToken? : ?string,
     branded? : boolean,
-    fundingSource : $Values<typeof FUNDING>
+    fundingSource : $Values<typeof FUNDING>,
+    buttonsArray : $ReadOnlyArray<?$Values<typeof CARD>>
```
Comment (ssono) [inline server/components/buttons/params.js:49]: Curious about the use of type `CARD`. In the other PR it looks like you are using Funding:  [
Screen Shot 2021-08-02 at 4 24 42 PM
](url)

Comment (ssono) [reply server/components/buttons/params.js:49]: Let's also make sure we consistent with our naming for both PRs

**Review by ssono — COMMENTED** [review]
Overall looks good. Do you need to add any tests? Also make sure that there is consistency with the names in both PRs and that they both work together before final merge

File: server/components/buttons/middleware.js (hunk 1 above)
Comment (lucasfrotam) [reply server/components/buttons/middleware.js]: Done!

File: server/components/buttons/params.js (hunk 2 above)
Comment (lucasfrotam) [reply server/components/buttons/params.js]: Done!

**Review by mnicpt — APPROVED** [review]

**Review by mnicpt — APPROVED** [review]

**Review by gregjopa — APPROVED** [review]

**Pull request merged**